File: poo/1. Calculating an Employee's Salary JS/extras.java
public class extras extends salary {
    private double dayTime;
    private double nightTime;
    private double holiDays;
    private double extrass;

    public extras(double dayTime, double nightTime, double holiDays, int salario) {
        super(1080292891, "Luis", "Duarte", salario);
        this.dayTime = dayTime;
        this.nightTime = nightTime;
        this.holiDays = holiDays;
    }
    public double getDayTime() {
        return dayTime;
    }
    public double getNightTime() {
        return nightTime;
    }
    public double getHoliDays() {
        return holiDays;
    }

    @Override
    public double payTotal(){
        this.extrass = (0.125 * (dayTime * this.getSalario())) + (0.175* (this.getSalario() * nightTime)) + (0.200 * (this.getSalario() * holiDays));
        return this.extrass;
    }
}

File: poo/1. Calculating an Employee's Salary/extras.php
<?php
class extras extends salary {
    private float $dayTime;
    private float $nightTime;
    private float $holiDays;
    private float $extrass;

    public function __construct(float $dayTime, float $nightTime, float $holiDays, int $salario) {
        parent::__construct(1080292891, "Juan", "Chala", $salario);
        $this->dayTime = $dayTime;
        $this->nightTime = $nightTime;
        $this->holiDays = $holiDays;
    }
    public function getDayTime(): float {
        return $this->dayTime;
    }
    public function getNightTime(): float {
        return $this->nightTime;
    }
    public function getHoliDays(): float {
        return $this->holiDays;
    }

    public function payTotal(): float {
        $this->extrass = (0.125 * ($this->dayTime * $this->salario)) + (0.175 * ($this->nightTime * $this->salario)) + (0.200 * ($this->holiDays * $this->salario));
        return $this->extrass;
    }
}
?>